src/components/kubectl/kubeconfig.ts

```diff
@@ -2,21 +2,59 @@ import * as vscode from 'vscode';
 import { fs } from '../../fs';
 import * as path from 'path';
 import * as yaml from 'js-yaml';
+import * as shelljs from 'shelljs';
 import { refreshExplorer } from '../clusterprovider/common/explorer';
-import { getActiveKubeconfig } from '../config/config';
+import { getActiveKubeconfig, getUseWsl } from '../config/config';
 
 interface Named {
     readonly name: string;
 }
 
+export function getKubeconfigPath(): { pathType: 'host'; hostPath: string } | { pathType: 'wsl'; wslPath: string }  {
```

Comment:
>{ pathType: 'host'; hostPath: string } | { pathType: 'wsl'; wslPath: string }  [](start = 37, length = 78)

Is there any reason not to use `HostKubeconfigPath | WslKubeconfigPath` here as well? If we can avoid it, there is no reason to redeclare these types here. :)